build-tests/heft-rspack-everything-test/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "heft-rspack-everything-test",
+  "description": "Building this project tests every task and config file for Heft when targeting the web browser runtime using Rspack",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "build": "heft build --clean",
+    "start": "heft build-watch",
+    "serve": "heft build-watch --serve",
+    "_phase:build": "heft run --only build -- --clean",
+    "_phase:build:ipc": "heft run-watch --only build -- --clean",
```

Comment:
Unless heft-rspack-plugin has a way to interrupt watch mode to ensure proper sequencing, probably best not to enable IPC mode for this project.